packages/plugins/src/plugins/eigenlayer/fetchers/withdrawal.ts

```diff
@@ -40,15 +40,20 @@ export const getWithdrawals = async (owner: Address, cache: Cache) => {
 
   const element = elementRegistry.addElementMultiple({
     label: 'Deposit',
-    name: 'Completable Withdrawals',
+    name: 'Withdrawals',
     tags: ['Withdrawal'],
   });
 
   shares.forEach((share) => {
+    const strategy = strategies?.find(
+      (str) =>
+        getAddress(str.strategyAddress) === getAddress(share.strategyAddress)
```

Comment:
```suggestion
       str.strategyAddress === share.strategyAddress
```
Given we're in a filter, is `getAddress()` really needed ?

Reply:
yes good catch I'm already doing it when I save the strategies in the cache so not needed here, updated !